### ISSUE TITLE
Missing Plutus script version compatiblity 

### ISSUE THREAD

**Issue by SeungheonOh** [IntersectMBO/formal-ledger-specifications#380]
In Babbage ledger spec, at Figure 3, it states about "allowedLanguages" that indicates what different versions of scripts are allowed in a transaction depending on some of features transaction uses. Specifically, the spec say V1 and V2 scripts are allowed when transaction doesn't have reference script nor inline datum; only V2 scripts are allowed when transaction has reference script or inline datum. 

Similar notation for Conway spec is needed so that developers can get a clear view on how a V3 script should be written to interact with older version scripts that are already deployed on chain.

**Comment by WhatisRT** [comment]
That's right, the big issue here is that we don't yet have support for multiple versions in the spec. There are a few more pressing issues and challenges associated with this, so it could be a while until we manage to fix this.

The general policy for Plutus versions is that we don't allow old Plutus versions to run in transactions that use new features.

**Comment by SeungheonOh** [comment]
Thanks! 

Will specification for this land before Conway lunch? Or will Conway nodes launch without any set spec over this topics

**Comment by SeungheonOh** [comment]
Okay, thanks for clarifying

**Comment by SeungheonOh** [comment]
https://github.com/IntersectMBO/formal-ledger-specifications/blob/1e627601fbcf60dfef47ec97e17c1d665822e1d4/src/Ledger/Utxow.lagda#L83

Haven been added. Thanks!
